store/actions/profiles.js

```diff
@@ -58,13 +60,18 @@ export function createProfile(
   image
 ) {
   return async (dispatch, getState) => {
-    const token = getState().auth.token;
-    const userId = getState().auth.userId;
-
+    // const token = getState().auth.token;
+    // const userId = getState().auth.userId;
```

Comment:
Please remove, don't just comment out unless there is a reason why we might put it back :)